### DISCUSSION TITLE
xmake分布式编译

### DISCUSSION THREAD

**Discussion by caxiejiang** [xmake-io/xmake#3014] [Q&A]
xmake尝试使用其他构建系统的情况是否支持分布式编译？比如xmake 调用调用androd.mk cmakelists的编译

**Comment by waruqi** [comment]
trybuild 模式只是一个辅助工具，不会支持这些，也不要期待更多，另外你应该到 cmake/meson 那边提 issues 才对，xmake 只是调用它们，辅助传参而已。

**Reply by caxiejiang** [reply]
确实~
谢谢解答
